_health-care/_js/components/Nav.jsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { connect } from 'react-redux';
```

Comment:
All of these changes are also from #1758 it looks like and shouldn't be included in this PR.